third-party/realdds/src/dds-participant.cpp

```diff
@@ -136,12 +135,12 @@ void dds_participant::init( dds_domain_id domain_id, std::string const & partici
     pqos.wire_protocol().builtin.discovery_config.leaseDuration = { 10, 0 };  // [sec,nsec]
 
     //Disable shared memory, use only UDP
+    //Disabling because sometimes, after improper destruction (e.g. stopping debug) the shared memory is not opened
```

Comment:
Next time, please please please add a space between the `//` and the text and keep everything looking similar.